api-metadata.json

```diff
@@ -422,10 +404,6 @@
       "minArgs": 0,
       "maxArgs": 0
     },
-    "getBrowserInfo": {
```

Comment:
This is the only change that should be kept. All other removals from this PR should be undone.

Reply:
Ohh Sorry! 
I've updated it.